mesh/nurbs.cpp

```diff
@@ -2001,23 +2001,23 @@ NURBSExtension::NURBSExtension(std::istream &input, bool spacing)
          input >> numSpacing;
          for (int j = 0; j < numSpacing; j++)
          {
-            int ki, spacingType, numIntParam, numDoubleParam;
-            input >> ki >> spacingType >> numIntParam >> numDoubleParam;
+            int ki, spacingType, numIntParam, numreal_tParam;
```

Comment:
This is fine, but `numRealParam` would be easier to read.

Reply:
Indeed. Changed to `Real`